lib/puppet/type/mailalias.rb

```diff
@@ -33,6 +30,11 @@ def should_to_s(value)
       end
     end
 
+    newproperty(:file) do
```

Comment:
@MikaelSmith my inclination is to be consistent and use `newproperty` for now and get the PR merged; we can try changing this value to a parameter in another pass.